docs/extend/plugins_graphdriver.md

```diff
@@ -5,8 +5,8 @@ graph driver for use with Docker engine. This is an alternative to using the
 built-in storage drivers, such as aufs/overlay/devicemapper/btrfs.
```

Comment:
Also, there should be metadata added for the docs (easiest to take other documents from the docker.github.io repo as example)